packages/dai-plugin-governance/src/utils/helpers.js

```diff
@@ -32,7 +31,7 @@ export const netIdtoSpockUrl = id => {
     case 42:
       return KOVAN_URL;
     default:
-      return LOCAL_URL;
```

Comment:
was your sense that there'd never really be a reason to use the plugin with a local instance of spock?

Reply:
Right, and this was a quick way to get the integration tests to work without a local instance of spock running.